### PR TITLE
ci: stop using OpenStack server groups in terraform

### PR DESCRIPTION
**Pull request by alexandre-allard**
**Component**: ci, tf

**Context**: 

**Summary**: removal of server groups in terraform

**Acceptance criteria**:  builds using terraform still work :)

### CODE REVIEW

File: eve/workers/openstack-multiple-nodes/terraform/nodes.tf [hunk 1]
```diff
@@ -1,18 +1,9 @@
-resource "openstack_compute_servergroup_v2" "all" {
```
Comment (slaperche-scality) [inline eve/workers/openstack-multiple-nodes/terraform/nodes.tf:1]: Commit message could mention **WHY** we remove them.

And as they seems useless (as we just remove them and be done with it), maybe explain what was their role/why it's not an issue to simply remove them.

Comment (alexandre-allard) [reply eve/workers/openstack-multiple-nodes/terraform/nodes.tf:1]: Yep, good point, I wanted to see first if it does not break anything in the CI, but it looks good.
Is the commit message sufficient or do you think we must put more information in it ?

**Review by slaperche-scality — DISMISSED** [review]

**Review by gdemonet — APPROVED** [review]

**Pull request merged**